js/app.js

```diff
@@ -4,6 +4,91 @@ $(document).ready( function() {
   // check to make sure JS is loaded
   console.log('JS is loaded!');
 
-  // your code here
+let searchInput = "Khalid";
```

Comment:
Hey Jimmy,

This code looks great and I'm really glad you reached out to other classmates to help guide you through the assignment - that's something we really encourage. Do you feel you understand this code and how it works, though? That's the most important part. If not, reach out, we're here to help! 

Matt